packages/core/demo/data/bar.ts

```diff
@@ -74,6 +74,35 @@ export const simpleBarOptions = {
 	},
 };
 
+// Simple bar with long labels
+export const simpleBarLongLabelData = [
+	{ group: "6591DA8668C339B1B39297C61091E320C35391AB7AFC15B469F96B8A2DD0C231", value: 65000 },
+	{ group: "347FEDE2F7403759069E5F84B65B49D2467D8914B5184738699259AA310EB0F9", value: 29123 },
+	{ group: "232D788298773BB389DBB8FCE44D3FB4E878879BE7AFB0B303BCE0D56EBB92E2", value: 35213 },
+	{ group: "58B01AADFA87E5547A218B3C6CE3AF07B8DF7BAB9E12BF60FD2BBB739C46B86E", value: 51213 },
+	{ group: "Qty", value: 16932 },
+];
+
+export const simpleBarLongLabelOptions = {
+	title: "Simple bar (truncated labels)",
+	axes: {
+		left: {
+			mapsTo: "value",
+		},
+		bottom: {
+			mapsTo: "group",
+			scaleType: "labels",
+		},
+	},
+	legend: {
+		truncation: {
+			type: "end_line",
```

Comment:
```suggestion
			type: TruncationTypes.END_LINE,
```

Reply:
I tried to `import { TruncationTypes } from "../../src/interfaces/enums";` in the bar.ts demo data file before. It worked fine in the storybook. However when I ran yarn build it was giving me error `[!] Error: Could not resolve '../../src/interfaces/enums' from dist/demo/data/bar.js`. This is quite strange. I am not sure why.